.github/workflows/broken-link-check.yml

```diff
@@ -1,22 +1,37 @@
+name: Links
+
 on:
+  repository_dispatch:
+  workflow_dispatch:
   schedule:
-  - cron: "0 0 * * 0"
-  repository_dispatch: # run manually
-    types: [check-link]
+    - cron: "0 0 * * 0"
 
 permissions: {}
 
-name: Broken Link Check
 jobs:
-  check:
-    name: Broken Link Check
+  linkChecker:
     runs-on: ubuntu-latest
-
     permissions:
       contents: read
       issues: write
-
-    if: github.repository == 'metal3-io/metal3-io.github.io'
+    if: github.repository == 'peppi-lotta/metal3-io.github.io'
     steps:
-    - name: Broken Link Check
-      uses: technote-space/broken-link-checker-action@b8332d945b97f8b52eb8d7d889a1e0e37106c1a9 # v2.3.2
+      - uses: actions/checkout@v4
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v2
+        with:
+          args: --user-agent "Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:59.0) Gecko/20100101 Firefox/59.0" --no-progress --accept 429 './**/*.md' './**/*.html' './**/*.rst'
```

Comment:
This seems to be the issue. It is parsing markdown files and rst files as is, and not actually the website.